org.eclipse.zest.core/src/org/eclipse/zest/core/widgets/Graph.java

```diff
@@ -223,7 +211,7 @@ public void removeSelectionListener(SelectionListener selectionListener) {
 	 * 
 	 * @return List of GraphModelNode objects
 	 */
-	public List getNodes() {
+	public List<GraphNode> getNodes() {
```

Comment:
from my experience with other generics users would work better if you you use here ? extends GraphNode